stargz/stargzify/stargzify.go

```diff
@@ -33,8 +34,11 @@ import (
 )
 
 var (
-	upgrade = flag.Bool("upgrade", false, "upgrade the image in-place by overwriting the tag")
-	flatten = flag.Bool("flatten", false, "flatten the image's layers into a single layer")
+	upgrade  = flag.Bool("upgrade", false, "upgrade the image in-place by overwriting the tag")
+	flatten  = flag.Bool("flatten", false, "flatten the image's layers into a single layer")
+	insecAll = flag.Bool("insecure", false, "allow connections to all registries using HTTP")
+	insecSrc = flag.Bool("insecure-src", false, "allow connections to the source registry using HTTP")
```

Comment:
Adding --insecure is okay, but the other two are just too many flags.

Let's overload the "input" and "output" non-flag arguments to mean that if they start with "http://", then they're HTTP. (And require --insecure)

Reply:
Thank you for your review. I fixed it to use the "input" and "output" arguments to specify HTTP registries.